src/libs/actions/User.js

```diff
@@ -531,8 +531,10 @@ function triggerNotifications(onyxUpdates) {
 
         const reportID = update.key.replace(ONYXKEYS.COLLECTION.REPORT_ACTIONS, '');
         const reportActions = _.values(update.value);
-        const sortedReportActions = ReportActionsUtils.getSortedReportActions(reportActions);
-        Report.showReportActionNotification(reportID, _.last(sortedReportActions));
+
+        // eslint-disable-next-line rulesdir/no-negated-variables
```

Comment:
What is this eslint rule for?

Reply:
It's a style guide thing. We're not supposed to use "negated" names so no `isNotAnExpense()` or `isNotOpen`. The   regex isn't great though so it thinks `isNotifiable` is negated

Reply:
😂 great explanation, thanks